fmriprep/__init__.py

```diff
@@ -23,8 +23,23 @@
     __description__,
     __longdesc__
 )
+from .due import due, Url, Doi
 
 import warnings
 
 # cmp is not used by fmriprep, so ignore nipype-generated warnings
 warnings.filterwarnings('ignore', r'cmp not installed')
+
+due.cite(
+    # Chicken/egg problem with Zenodo here regarding DOI.  Might need
```

Comment:
What is the status regarding this? There is a DOI for the Zenodo entry now, right?